code/modules/medical/genetics/bioEffects/hemochromia.dm

```diff
@@ -0,0 +1,278 @@
+ABSTRACT_TYPE(/datum/bioEffect/hemochromia)
```

Comment:
Move this to the line above `/datum/bioEffect/hemochromia` definition so it's easier to spot, please.

Reply:
Done.